test/release/examples/benchmarks/hpcc/HPCCProblemSize.chpl

```diff
@@ -6,7 +6,7 @@ module HPCCProblemSize {
   //
   // Use the standard modules for reasoning about Memory and Types
   //
-  use Memory.Diagnostics, Types;
+  use MemDiagnostics;
```

Comment:
Hmm, I need to check whether we have a policy regarding the usage of unstable features in the release.